packages/labs/ssr-dom-shim/src/lib/events.ts

```diff
@@ -0,0 +1,567 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * This implementation is not fully spec compliant (e.g. validation),
+ * but should work well enough for our use cases.
+ */
+
+type EventTargetInterface = EventTarget;
+
+const isCaptureEventListener = (
+  options: undefined | AddEventListenerOptions | boolean
+) => (typeof options === 'boolean' ? options : options?.capture ?? false);
+
+// Shim the global EventTarget object
```

Comment:
EventTarget already exists, I think we can extend it. See https://github.com/lit/lit/pull/2309/files#diff-faf9adf12117ea971af406ace7fdbb87f6c19ae279972892b67b5c0c520d61a4R40

Reply:
Yes, but the behavior is different, both with the capture phase and the property states (that are patched in this implementation).
Also it would fail in the empty VM mode.